Packs/UrlScan/Integrations/UrlScan/UrlScan.yml

```diff
@@ -40,6 +40,11 @@ configuration:
   name: is_public
   required: false
   type: 8
+- defaultvalue: 'true'
+  display: Create Relationships
+  name: relationships
+  required: false
+  type: 8
 description: Urlscan.io reputation
 display: urlscan.io
```

Comment:
general comment, new added outputs of relationships should be added to the relevant commands.